terraform-static-analysis/entrypoint.sh

```diff
@@ -123,9 +123,9 @@ run_tflint(){
         echo "Excluding the following checks: ${INPUT_TFLINT_EXCLUDE}"
         readarray -d , -t tflint_exclusions <<< $INPUT_TFLINT_EXCLUDE
         tflint_exclusions_list=( "${tflint_exclusions[@]/#/--disable-rule=}" )
-        tflint --config $tflint_config ${tflint_exclusions_list[@]} ${terraform_working_dir} 2>&1
+        tflint --config $tflint_config --disable-rule ${tflint_exclusions_list[@]} --chdir ${terraform_working_dir} 2>&1
```

Comment:
```suggestion
        tflint --config $tflint_config ${tflint_exclusions_list[@]} --chdir ${terraform_working_dir} 2>&1
```
Just looking in the line above, it looks like the options is passed as part of the array